index.html

```diff
@@ -4,9 +4,56 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500" rel="stylesheet">
   <title>Hello Front-End</title>
 </head>
 <body>
-  <h1>Hello Front-End</h1>
+  <header>
+	  <!-- <h1>Hello Front-End</h1>  	 -->
+  </header>
+  <main>
+  	<div class="block">
+  		<h2>Meet Guidebooks</h2>
+  		<p>Discover hundrends of local spots recommended by AirBnB hosts</p>
+  		<div class="imgBlock">
+  			<!-- <label>San Francisco</label> -->
+  			<div class="imgDiv">
+  				<h4>San Francicso</h4>
+  				<img src="img/san-francisco.jpg">
+  			</div>
```

Comment:
An alternate approach to using an `<img>` tag here would be to set that image as the `background-image` of the `<div>`. Then the only HTML content of the `<div>` would be the `<p>`, which you can center horizontally and vertically using flexbox.